.gitlab/validate-logs-intgs/validate_log_intgs.py

```diff
@@ -7,8 +7,8 @@
 import yaml
 import os
 
-LOGS_BACKEND_INTGS_ROOT = os.path.abspath(os.environ['LOGS_BACKEND_INTGS_ROOT'])
-INTEGRATIONS_CORE = os.path.abspath(os.environ['INTEGRATIONS_CORE_ROOT'])
+LOGS_BACKEND_INTGS_ROOT = os.environ['LOGS_BACKEND_INTGS_ROOT']
+INTEGRATIONS_CORE = os.environ['INTEGRATIONS_CORE_ROOT']
```

Comment:
Is this change needed?

Reply:
IMO it's better to completely control the paths in the environment variables here https://github.com/DataDog/integrations-core/pull/10275/files#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7R40